GLMakie/src/drawing_primitives.jl

```diff
@@ -499,12 +513,12 @@ function mesh_inner(screen::Screen, mesh, transfunc, gl_attributes, space=:data)
     return draw_mesh(screen, mesh, gl_attributes)
 end
 
-function draw_atomic(screen::Screen, scene::Scene, meshplot::Mesh)
-    return cached_robj!(screen, scene, meshplot) do gl_attributes
-        t = transform_func_obs(meshplot)
```

Comment:
I think the others shouldn't use `x` and instead use a better name instead of changing everything to x...

Reply:
i'd be fine to use the same name, e.g. `plot` or something like that... But that I started using `x` for the plot at some point was really bad I think :D

Reply:
I'll change that yes, better for readability.